stats/gRPC.md

```diff
@@ -119,7 +119,7 @@ This way users can configure views to correlate incoming with outgoing requests.
 
 | View name                               | Measure suffix | Aggregation  | Tags suffix                  |
 |-----------------------------------------|----------------|--------------|------------------------------|
-| grpc.io/client/latency_by_server_method | latency        | distribution | client_method, server_method |
+| grpc.io/client/latency_by_server_method | server_latency | distribution | client_method, server_method |
```

Comment:
This is the client roundtrip_latency most likely

Reply:
Fixed